decimal.go

```diff
@@ -925,13 +939,13 @@ func (d *Decimal) UnmarshalJSON(decimalBytes []byte) error {
 
 	str, err := unquoteIfQuoted(decimalBytes)
 	if err != nil {
-		return fmt.Errorf("Error decoding string '%s': %s", decimalBytes, err)
+		return err
 	}
 
 	decimal, err := NewFromString(str)
 	*d = decimal
 	if err != nil {
-		return fmt.Errorf("Error decoding string '%s': %s", str, err)
+		return err
```

Comment:
Do you think we should keep the error message unmodified if we consider it as a part of API?

Reply:
We can regard master as development branch and pump major version once we finished all breaking changes for dexon usage.